src/apprt/gtk/App.zig

```diff
@@ -1073,23 +1066,50 @@ fn loadCustomCss(self: *App) Allocator.Error!void {
             .optional => |path| .{ path, true },
             .required => |path| .{ path, false },
         };
-        std.fs.accessAbsolute(path, .{}) catch |err| {
+        const file = std.fs.openFileAbsolute(path, .{}) catch |err| {
             if (err != error.FileNotFound or !optional) {
                 log.err("error opening gtk-custom-css file {s}: {}", .{ path, err });
             }
             continue;
         };
+        defer file.close();
 
-        const provider = c.gtk_css_provider_new();
+        log.info("loading gtk-custom-css path={s}", .{path});
+        var buf_reader = std.io.bufferedReader(file.reader());
+        const contents = try buf_reader.reader().readAllAlloc(
```

Comment:
I think that the buffered reader is unnecessary.

Reply:
switched to a non-buffered reader